src/components/VideoPlayer/VideoPlayerContainer.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext, useEffect } from 'react';
+import styled from 'styled-components';
+import SearchContext from '../../providers/SearchContext';
+import RelatedVideoList from './RelatedVideoList';
+
+const StyledVideoPlayerContainer = styled.div`
+  display: flex;
+`;
+
+const VideoPlayerContainer = ({ videoId }) => {
+  const { videos } = useContext(SearchContext);
+
+  useEffect(() => {
+    /* global YT */
+    /* eslint no-undef: "error" */
+      window.player = new YT.Player('player', {
+        height: '50%', 
+        width: '60%',
+        videoId,
+      });
+  });
```

Comment:
It might be a good idea to add the `videoId` as a dependency for this `useEffect` so this code its executed only when the `videoId` changes.

```suggestion
  useEffect(() => {
    /* global YT */
    /* eslint no-undef: "error" */
      window.player = new YT.Player('player', {
        height: '50%', 
        width: '60%',
        videoId,
      });
  }, [videoId]);
```